android/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
+    <uses-permission android:name="android.permission.VIBRATE" />
```

Comment:
I don't really know. I've followed the instructions to set it up at https://v5.rnfirebase.io/docs/v5.x.x/notifications/android and that's all they require. I think android conflates notifications with something else, so that everyone grants them, but that's just me seeing evil everywhere :)